spec/models/principal_spec.rb

```diff
@@ -19,24 +19,22 @@
 
 require File.dirname(__FILE__) + '/../spec_helper'
 
-describe Principal, :type => :model do
-  describe "ATTRIBUTES" do
+describe Principal, type: :model do
+  describe 'ATTRIBUTES' do
     before :each do
-
     end
 
     it { is_expected.to have_many :principal_roles }
     it { is_expected.to have_many :global_roles }
-
   end
 
-  describe "WHEN deleting a principal" do
+  describe 'WHEN deleting a principal' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.